test/functional/wallet_upgradewallet.py

```diff
@@ -92,6 +90,32 @@ def dumb_sync_blocks(self):
             v16_3_node.submitblock(b)
         assert_equal(v16_3_node.getblockcount(), to_height)
 
+    def test_upgradewallet(self, wallet, previous_version, requested_version=None, expected_version=None):
```

Comment:
nit 99d56e357159c7154f69f28cb5587c5ca20d6594:

```suggestion
    def test_upgradewallet(self, wallet, *, previous_version, requested_version, expected_version=None):
```

Could force named args and pass requested_version in all cases for clarity. Only one call needs to be adjusted, I think.

Reply:
> nit [99d56e3](https://github.com/bitcoin/bitcoin/commit/99d56e357159c7154f69f28cb5587c5ca20d6594):
> 
> Could force named args and pass requested_version in all cases for clarity. Only one call needs to be adjusted, I think.

Good idea.

Reply:
> nit [99d56e3](https://github.com/bitcoin/bitcoin/commit/99d56e357159c7154f69f28cb5587c5ca20d6594):
> 
> Could force named args and pass requested_version in all cases for clarity. Only one call needs to be adjusted, I think.

Thanks! Good idea.

Reply:
Done in e915a2a2c87dad5beb68a175b21a07136bcc21ca